MdePkg/Library/BaseLib/CheckSum.c

```diff
@@ -762,3 +762,74 @@ CalculateCrc32c (
 
   return ~Crc;
 }
+
+// MU_CHANGE [START] - Add CRC16-CCITT-FALSE implementation

```

Comment:
Since we are here, can we please add the source of this table into the comment? I put it in the commit message originally, but I think in the code would be better reflecting the state of the implementation:

The lookup table is inherited from `[https://crccalc.com/`](https://crccalc.com/%60)

Reply:
Yes, I added it, and it will be in the next push.